### PR TITLE
Added Navbar footer

### PR DESCRIPTION
**Pull request by Ankur9669**
Files to Check:
1. App.js
2. Navbar.js
3. Footer.js
4. PrimaryButton.js
5. HomePage.js

# Navbar
image

# Footer
image

### CODE REVIEW

File: fitbit-react/src/App.css [hunk 1]
```diff
@@ -1,38 +1,24 @@
...
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+:root{
+    --light-blue: #f2fbff;
+    --color-black: #000000;
+    --color-white: #ffffff;
+    --navbar-shadow: #b5b5b5;
+    --gray-color: gray;
```
Comment (bhtibrewal) [inline fitbit-react/src/App.css]: change it to hex for consistency

File: fitbit-react/src/Components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./footer.css";
+import { BsGithub } from "react-icons/bs";
+import { BsLinkedin } from "react-icons/bs";
+import { BsTwitter } from "react-icons/bs";
```
Comment (bhtibrewal) [inline fitbit-react/src/Components/Footer/Footer.js]: Change: you can write these in one line

File: fitbit-react/src/Components/Navbar/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { GrMenu } from "react-icons/gr";
+import "./navbar.css";
+import SearchBar from "./SearchBar";
+import PrimaryButton from "../Buttons/PrimaryButton";
+import Logo from "./Logo";
+import NavbarIcons from "./NavbarIcons";
```
Comment (bhtibrewal) [inline fitbit-react/src/Components/Navbar/Navbar.js]: Suggestion: think about a way to reduce the no of lines as in future its going to too long

**Review by bhtibrewal — APPROVED** [review]
Looks good to me, just a suggestion try thinking more modular in terms of expandability and reducing lines of code

File: fitbit-react/src/Components/Footer/Footer.js (hunk 2 above)
Comment (Ankur9669) [reply fitbit-react/src/Components/Footer/Footer.js]: Sure this is a good catch

File: fitbit-react/src/Components/Navbar/Navbar.js (hunk 3 above)
Comment (Ankur9669) [reply fitbit-react/src/Components/Navbar/Navbar.js]: I don't think there is any way to reduce this🤔

File: fitbit-react/src/Components/Navbar/Navbar.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+function Navbar() {
```
Comment (amantulsyan35) [inline fitbit-react/src/Components/Navbar/Navbar.js:5]: can change is to ES6 syntax ==>  const Navbar = () => {}

File: fitbit-react/src/App.css (hunk 1 above)
Comment (Ankur9669) [reply fitbit-react/src/App.css]: Done

File: fitbit-react/src/Components/Navbar/Navbar.js (hunk 4 above)
Comment (Ankur9669) [reply fitbit-react/src/Components/Navbar/Navbar.js:5]: Ya i can, but i like my components to be in this manner

File: fitbit-react/src/Pages/Homepage/Homepage.js [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Footer from "../../Components/Footer/Footer";
+import Navbar from "../../Components/Navbar/Navbar";
```
Comment (yaswanthmyneni) [inline fitbit-react/src/Pages/Homepage/Homepage.js]: changes: you can import all components to index.js file and import from that file, so you can reduce the lines of import.

File: fitbit-react/src/Components/Navbar/NavbarIcons.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { BsFillHeartFill } from "react-icons/bs";
+import { AiOutlineHeart } from "react-icons/ai";
+import { AiFillHeart } from "react-icons/ai";
+import { FaShoppingCart } from "react-icons/fa";
```
Comment (yaswanthmyneni) [inline fitbit-react/src/Components/Navbar/NavbarIcons.js]: changes: same thing you can do here too. As all are coming from same folder(react-icons), you can create an icon folder in assets and create index.js and import and export all the icons you need in your project from there itself, so from 2 to 5 lines here becomes to one line.

**Review by yaswanthmyneni — APPROVED** [review]
You can merge after making changes. The way you wrote onClick function in button component is new for me, it will help me in my projects.

**Review by Shivani-1524 — APPROVED** [review]
other than the comments by the others, it looks good to me.

File: fitbit-react/src/Components/Buttons/PrimaryButton.js [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+// Here onClick prop is a function which defines what to do on a button click
+// If onCLick is not passed it takes a function which does nothing
+function PrimaryButton({ buttonText, onClick = () => {} }) {
```
Comment (gauravsinhaweb) [inline fitbit-react/src/Components/Buttons/PrimaryButton.js:5]: not a proper way to pass a function on onClick

File: fitbit-react/src/Components/Buttons/PrimaryButton.js [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+  return (
+    <button className="button button-primary" onClick={onClick()}>
```
Comment (gauravsinhaweb) [inline fitbit-react/src/Components/Buttons/PrimaryButton.js]: not a proper way to pass a function on onClick

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: fitbit-react/src/Components/Buttons/PrimaryButton.js (hunk 7 above)
Comment (Ankur9669) [reply fitbit-react/src/Components/Buttons/PrimaryButton.js:5]: Done

File: fitbit-react/src/Components/Buttons/PrimaryButton.js (hunk 8 above)
Comment (Ankur9669) [reply fitbit-react/src/Components/Buttons/PrimaryButton.js]: Done

File: fitbit-react/src/Components/Navbar/NavbarIcons.js (hunk 6 above)
Comment (Ankur9669) [reply fitbit-react/src/Components/Navbar/NavbarIcons.js]: Done

File: fitbit-react/src/Components/Navbar/Navbar.js (hunk 3 above)
Comment (Ankur9669) [reply fitbit-react/src/Components/Navbar/Navbar.js]: Done

**Pull request merged**